domain/schema/model/sql/0005-model.sql

```diff
@@ -73,3 +73,17 @@ SELECT
 FROM constraint_zone AS cz
 JOIN "constraint" AS c ON cz.constraint_uuid = c.uuid
 JOIN model_constraint AS mc ON c.uuid = mc.constraint_uuid;
+
+-- This table is best effort to track the life of a model. The real location
+-- of the model life is in the controller database. This is just a facsimile
+-- of the model life.
```

Comment:
todo: add text on why this table is here.

Reply:
Does that not state that in code comment? Anyone can look at the life of the model and see if it's dying.